recipes/mo-kwargs/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "mo-kwargs" %}
+{% set version = "3.93.20259" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mo-kwargs-{{ version }}.tar.gz
+  sha256: c7802ad3211f25536a73ae5775efa398ca59f5b23551957f79e53d4ccf179d2e
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=2.7
```

Comment:
should this be 3.6? it may not matter. I am not 100% sure.

Reply:
No, this project actually supports 2.7